### ISSUE TITLE
Convert SiteEncryption Key to generate a random 32 character value

### ISSUE THREAD

**Issue by digioz** [digioz/digioznetportal#16]
Right now the Site Encryption Key is hard coded in the seed data. This needs to be changed to generate a random 32 character string during Data Seed to make it unique from site to site.

**Comment by sartul** [comment]
We can change the line no 25 in digioz.Portal.Web.Data.Migrations.MigrateOtherDataSeed to get a random 32 character key with some function rather than passing hardcoded key.
The function can be simple c# function which will accept length as parameter and will generate the key as length specified in function call.
Let me know your views.

**Comment by digioz** [comment]
Yep, that's exactly what I was thinking too. We actually already had a function to generate random strings, which I reused. I also updated the seed data to encrypt the default SMTPPassword password with the new randomly generated site encryption key. 

Take a look here and let me know if you agree so we can mark this as resolved:

https://github.com/digioz/digioznetportal/commit/561fe81b23aaa7a65eeb1d878b27f072f2ccae08
